Bench.php

```diff
@@ -311,6 +315,43 @@ public function getLongest()
     }
 
     /**
+     * Pause all Marks.
+     *
+     * @return  array
+     */
+    public function pause()
+    {
+        $runningMarks = [];
+
+        foreach ($this as $mark) {
+            if ($mark->isRunning()) {
```

Comment:
Mmmh, and  false === $mark->isPause()
I didn't realized oneBis is running again, but it's manually paused.